test/unit/maintenance/__snapshots__/maintenance.test.ts.snap

```diff
@@ -1,105 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Maintenance Frontend Atlassian Connect should return Atlassian Connect JSON in maintenance mode 1`] = `
-Object {
-  "apiMigrations": Object {
-    "gdpr": false,
-  },
-  "apiVersion": 1,
-  "authentication": Object {
-    "type": "jwt",
-  },
-  "description": "Connect your code and your project with ease.",
-  "lifecycle": Object {
-    "disabled": "/jira/events/disabled",
-    "enabled": "/jira/events/enabled",
-    "installed": "/jira/events/installed",
-    "uninstalled": "/jira/events/uninstalled",
-  },
-  "modules": Object {
-    "adminPages": Array [
-      Object {
-        "conditions": Array [
-          Object {
-            "condition": "addon_property_exists",
-            "invert": true,
-            "params": Object {
-              "objectKey": "has-repos",
-              "propertyKey": "configuration",
-            },
-          },
-          Object {
-            "condition": "user_is_admin",
-          },
-        ],
-        "key": "gh-addon-admin",
-        "location": "admin_plugins_menu/gh-addon-admin-section",
-        "name": Object {
-          "value": "Configure integration",
-        },
-        "url": "/jira/configuration",
-      },
-    ],
-    "jiraDevelopmentTool": Object {
-      "application": Object {
-        "value": "GitHub",
-      },
-      "capabilities": Array [
-        "branch",
-        "commit",
-        "pull_request",
-      ],
-      "key": "github-development-tool",
-      "logoUrl": "https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png",
-      "name": Object {
-        "value": "GitHub",
-      },
-      "url": "https://github.com",
-    },
-    "postInstallPage": Object {
-      "conditions": Array [
-        Object {
-          "condition": "addon_property_exists",
-          "invert": true,
-          "params": Object {
-            "objectKey": "has-repos",
-            "propertyKey": "configuration",
-          },
-        },
-        Object {
-          "condition": "user_is_admin",
-        },
-      ],
-      "key": "github-post-install-page",
-      "name": Object {
-        "value": "GitHub Configuration",
-      },
-      "url": "/jira/configuration",
-    },
-    "webSections": Array [
-      Object {
-        "key": "gh-addon-admin-section",
-        "location": "admin_plugins_menu",
-        "name": Object {
-          "value": "GitHub",
-        },
-      },
-    ],
-  },
-  "scopes": Array [
-    "READ",
-    "WRITE",
-    "DELETE",
-  ],
-  "vendor": Object {
-    "name": "Atlassian",
-    "url": "https://atlassian.com",
-  },
-}
-`;
-
-exports[`Maintenance Frontend Maintenance should return 503 for any frontend routes 1`] = `Object {}`;
-
```

Comment:
Where'd this all go?  Your tests are probably not working as it should as the snapshot should be the same.

Reply:
No clue why GitHub shows this as deleted. It's still there: https://github.com/atlassian/github-for-jira/blob/feature-flags/test/unit/maintenance/__snapshots__/maintenance.test.ts.snap